qcfractal/alembic/env.py

```diff
@@ -29,6 +32,13 @@
 # my_important_option = config.get_main_option("my_important_option")
 # ... etc.
 
+# Overwrite the ini-file sqlalchemy.url path
+fractal_conf = FractalConfig()
+fractal_conf = FractalConfig(**yaml.load(fractal_conf.config_file_path.read_text(),
```

Comment:
Sometimes the `base-folder` will change so this needs to be able to be parametrized from the CLI side.